src/main/java/com/android/volley/toolbox/DiskBasedAsyncCache.java

```diff
@@ -225,4 +312,42 @@ private void deleteFile(File file) {
         }
         initializeIfRootDirectoryDeleted();
     }
+
+    /** Empty onCompleteCallback when no operation is needed in the callback. */
+    private OnCompleteCallback emptyCallback =
+            new OnCompleteCallback() {
+                @Override
+                public void onComplete() {
+                    // do nothing
+                }
+            };
```

Comment:
This is a variable, so I'd suggest to move it at the top. Also, given that it's a constant, could you make it `static final`?

Reply:
I ended up just getting rid of this